.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -526,9 +538,15 @@ stages:
         x64:
           baseImage: debian
           artifactSuffix: linux-x64
+          readyToRun: false
         alpine:
           baseImage: alpine
           artifactSuffix: linux-musl-x64
+          readyToRun: false
+        x64-r2r:
+          baseImage: debian
+          artifactSuffix: linux-x64-r2r
+          readyToRun: true
```

Comment:
I think this is probably making things more complicated than they need to be 🤔 Rather than trying to be everything to everyone, you could add your own packaging stage, which makes it easier to build the AWS layers? 

If you do that you can potentially keep everything simpler and faster - you can download only the artifacts you need, and combine them as necessary (e.g. x64 + arm64)? I think that's ultimately what we should be aiming for, and it would avoid making this stage (which is already v complex) more confusing 😃

Reply:
Switch to make it its own step

Reply:
I updated it to create its own step for `x64` and `arm64` and then the GitLab scripts downloads the specifics binaries from there